frontend/lib/search/location_button.dart

```diff
@@ -64,28 +64,65 @@ class _LocationButtonState extends State<LocationButton> {
         ),
       );
     } else {
-      return Container();
+      return Container(
+        alignment: Alignment.bottomCenter,
+        padding: const EdgeInsets.all(10),
+        child: FloatingActionButton.extended(
+          heroTag: 'fab_search_view',
+          backgroundColor: Theme.of(context).colorScheme.surfaceVariant,
+          elevation: 1,
+          onPressed: () => _sortAlphabetically(settings),
+          label: Text(
+            t.search.findAlphabetically,
+            style: TextStyle(color: Theme.of(context).hintColor),
+          ),
+        ),
+      );
     }
   }
 
+  Future<void> _showFeatureDisabled() async {
+    final messengerState = ScaffoldMessenger.of(context);
+    final t = context.t;
+    messengerState.showSnackBar(
+      SnackBar(
```

Comment:
The snackbar is currently shown on top of the FAB, let me know if I should adjust this.

Reply:
thats okay i think. Can you give me a hint how i can test this snackbar message

Reply:
Okay. Sure, block the location permission and press the FAB :)

Reply:
not its okay. its just temporary

Reply:
@f1sh1918 what do you mean by that?

Reply:
the snackbar is just showing up temporary for some seconds. so its okay for me